crates/libtiny_tui/src/messaging.rs

```diff
@@ -46,9 +46,21 @@ pub(crate) struct Timestamp {
 }
 
 impl Timestamp {
+    /// The width of the timestamp plus a space
+    pub(crate) const WIDTH: usize = 6;
     fn stamp(self, msg_area: &mut MsgArea) {
-        msg_area.set_style(SegStyle::SchemeStyle(SchemeStyle::Timestamp));
-        msg_area.add_text(&format!("{:02}:{:02} ", self.hour, self.min));
+        msg_area.add_text(
+            &format!("{:02}:{:02} ", self.hour, self.min),
+            SegStyle::Timestamp,
+        );
+    }
+
+    /// Inserts a blank space that is the size of a timestamp
+    fn blank(msg_area: &mut MsgArea) {
+        msg_area.add_text(
+            &format!("{:^width$}", "", width = Timestamp::WIDTH),
```

Comment:
This allocates a `String`, but it's actually a static string. I think we can't create `&'static str` using `WIDTH`, but having a const/static string next to `WIDTH`, with documentation that the length of the string should be `WIDTH` should be fine.